docs/program/apis/disconnect.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Client Sessions and Robot Networking"
+linkTitle: "Network Disconnection"
+weight: 20
+type: "docs"
+description: "How a Viam robot handles losing its connection to the internet."
```

Comment:
```suggestion
description: "Whan a robot loses its connection to the internet, all client session will timeout and end by default."
```
^ this sentence provides a bit of information which is then elaborated on in the text. The current sentence describes what this section covers without providing an answer. I think the former is better

Reply:
LGTM, implemented-- unfortunately I didn't rename properly with git and can't apply inline but copy/paste applied